database/engine/metadata_log/logfile.c

```diff
@@ -552,7 +552,7 @@ static int scan_metalog_files(struct metalog_instance *ctx)
         .obsolete = 0,
         .started_t = INVALID_TIME,
         .next = NULL,
-        .version = 0,
+        .version = 9999,
```

Comment:
Shouldn't `.version = ` be also set in `receiver.c`?

Reply:
It is set right before calling the streaming parser

Reply:
Whoops, yes. I just saw `.version = 0` and didn't realize it's being overwritten.